lib/index.js

```diff
@@ -37,19 +37,41 @@ function addSnack() {
   })
 }
 
+function addMeal(tableID) {
+  var selectedFoods = $(':checked').parent().parent();
+  Array.from(selectedFoods).forEach(function(foodRow){
+    var name = $(foodRow).children('.food-name').text();
+    var calories = $(foodRow).children('.food-calories').text();
+    addMealRow(tableID, name, calories);
+  })
+}
 
-function populateCheckBox() {
-  $("tbody#diary-food-list tr").each(function() {
-    console.log(this.parent);
-    $(this).prepend("<td><input type='checkbox'/></td>");
-  });
+function addMealRow(tableID, name, calories) {
+  var row = "<tr class='food-row'>" +
+            "<td class='food-name'>" + 
+            name + 
+            "</span></td> <td class='food-calories'>" + 
+            calories + 
+            "</span></td> <td class='food-delete'><button>-</button></td>";
+  $(tableID + '-list').prepend(row);
+  $('#name-field input, #calories-field input').val('')
 }
 
-function addMeal(buttonType) {
-  var selectedFoods = $(':checked').parent().parent();
-  var mealType = $(buttonType).text();
-  Array.from(selectedFoods).forEach(function(foodItem){
-    console.log(foodItem.children);
+function addDiaryRow(name, calories) {
+  var row = "<tr class='food-row'>" +
+            "<td><input type='checkbox'/></td>" +
+            "<td class='food-name'>" + 
+            name + 
+            "</span></td> <td class='food-calories'>" + 
+            calories
+  $('#diary-food-list').prepend(row);
+  $('#name-field input, #calories-field input').val('');
```

Comment:
you are also doing this same thing on line 57 - can we encapsulate this in its own function?